.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please also add .DS_Store to this list

You've ignored .idea, but it's actually too late because it has already been added to the repo. The top answer here is what you need: https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign